### PR TITLE
Add miraheze specific configs for discord hook

### PR DESCRIPTION
**Pull request by paladox**

### CODE REVIEW

File: modules/mediawiki/templates/PrivateSettings.php [hunk 1]
```diff
@@ -24,3 +24,7 @@
 
 // Matomo Token
 $wgMatomoAnalyticsTokenAuth = "<%= @matomotoken %>";
+
```
Comment (Reception123) [inline modules/mediawiki/templates/PrivateSettings.php]: Let's have a // Extension:DiscordNotifications hooks 

here

File: modules/mediawiki/templates/PrivateSettings.php [hunk 2]
```diff
@@ -24,3 +24,7 @@
...
+$wmgMetaMirahezeDiscordHook = "<%= @meta_discord_hook %>";
+
+$wmgWikiMirahezeDiscordHook = <%= @wiki_discord_hook %>;
```
Comment (Reception123) [inline modules/mediawiki/templates/PrivateSettings.php]: wouldn't there be "" as for the meta_discord_hook?

**Comment by JohnFLewis** [comment]
Why are you adding a private wiki settings file for LocalWiki? Don’t. Please don’t.

One is enough.

**Comment by paladox** [comment]
Because the hook url has to stay private and PS.php is loaded far too early.

**Comment by JohnFLewis** [comment]
No?

$wmg as an array, load the array variable.

You are not making another private file for no reason.

File: modules/mediawiki/templates/PrivateSettings.php (hunk 1 above)
Comment (paladox) [reply modules/mediawiki/templates/PrivateSettings.php]: done

**Pull request merged**